index.js

```diff
@@ -0,0 +1,167 @@
+const express = require('express');
+const helmet = require('helmet');
+const knex= require('knex');
+
+const knexConfig = require('./knexfile');
+const db = knex(knexConfig.development); 
+
+const server = express();
+
+server.use(helmet());
```

Comment:
use morgan or any other logging library they are helpful in both development and production